05_visualize_gaze_velocity.ipynb

```diff
@@ -113,8 +113,8 @@
    "source": [
```

Comment:
<p>These graphs look much better now! Could you please make a commit that does not show the deprecation warning?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pupil-labs/pupil-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Yes i can do this :) Do i have to start a new pull request now ?